GraphConsoleAppV3/Constants.cs

```diff
@@ -2,12 +2,19 @@
 {
     internal class Constants
```

Comment:
>Constants [](start = 19, length = 9)

Make 3 sections here
Constants - never to be modified
AppOnly mode related constants
App+User mode related constants

Make sure you name them consistently. e.g. Here I see TenantName and TenantNameAppOnly - i guess TenantName will be used for app+user. But i also see ClientId and ClientIdForUserAuthn - i guess here ClientId  will be used for app only mode.